spec/defines/host_spec.rb

```diff
@@ -19,12 +23,39 @@
           "class { '::dhcp': interfaces => ['eth0']}"
         end
 
-        let(:fragment_name) do
-          'dhcp.hosts+10_myhost.hosts'
+        let(:expected_content) do
+          <<~CONTENT
+            host myhost {
+              hardware ethernet   01:02:03:04:05:06;
+              fixed-address       10.0.0.100;
+              ddns-hostname       "myhost";
+            }
+          CONTENT
+        end
+
+        it do
+          is_expected.to contain_concat__fragment(fragment_name).with_content(expected_content)
+        end
+      end
+
+      describe 'comment parameter' do
+        let :params do {
+          :ip      => '10.0.0.100',
+          :mac     => '01:02:03:04:05:06',
+          :comment => 'a useful comment',
+        } end
+
+        let :facts do
+          facts
+        end
+
+        let :pre_condition do
+          "class { '::dhcp': interfaces => ['eth0']}"
         end
```

Comment:
Both this `let(:pre_condition)` and `let(:facts)` can be shared by both `describe` blocks by moving them up a level.

Reply:
I've normalized the file.